docs/src/main/asciidoc/getting-started-reactive.adoc

```diff
@@ -594,6 +594,25 @@ This resource returns `Uni` and `Multi` instances based on the result produced b
 
 The previous example uses a _service_ provided by Quarkus.
 Also, you can use Vert.x clients directly.
+
+Firstly, you need to activate the `vertx` extension by executing the following command.
```

Comment:
Maybe we should use the NOTE instead?

```
NOTE: First of all, make sure the `quarkus-vertx` extension is present. If not, activate the extension by executing the following command: ...
```

But otherwise +1.

Reply:
Changed as suggested.